conda_build/source.py

```diff
@@ -112,13 +111,16 @@ def git_source(meta, recipe_dir):
 
     # update (or create) the cache repo
     if isdir(cache_repo):
-        check_call([git, 'fetch'], cwd=cache_repo)
+        output = check_output([git, 'fetch'], cwd=cache_repo, stderr=STDOUT).decode()
+        logger.debug(output)
     else:
```

Comment:
By changing `check_call()` into `check_output()`, you are hiding the output (which is the intention here, obviously).  But if something goes wrong in the call, I want to see what the output from git was just before it failed.  I think this would be sufficient:

```python
try:
    output = check_output([git, 'fetch'], cwd=cache_repo, stderr=STDOUT).decode()
    logger.debug(output)
except subprocess.CalledProcessError as e:
    print(ex.output)
    raise # Or maybe sys.exit()?
```